packages/core/src/walk.ts

```diff
@@ -185,7 +198,7 @@ export function walkDocument<T>(opts: {
 
       const anyEnterVisitors = normalizedVisitors.any.enter;
       const currentEnterVisitors = anyEnterVisitors.concat(
-        normalizedVisitors[type.name]?.enter || []
+        (normalizedVisitors[type.name]?.enter as NormalizeVisitor<VisitorNode<any>[]>) || []
```

Comment:
Can we use `unknown` instead of `any` here ?
```suggestion
        (normalizedVisitors[type.name]?.enter as NormalizeVisitor<VisitorNode<unknown>[]>) || []
```